scripts/clean.sh

```diff
@@ -26,6 +26,10 @@ rm -rdf packages/flutter_nekoton_bridge/ios/Frameworks/nekoton_bridge*.zip
 rm -rdf packages/flutter_nekoton_bridge/macos/Frameworks/NekotonBridge.xcframework
 rm -rdf packages/flutter_nekoton_bridge/macos/Frameworks/nekoton_bridge*.zip
 
+# Remove js-related files
+rm -rdf packages/flutter_nekoton_bridge/js/node_modules/
+find packages/flutter_nekoton_bridge/assets/js/ -type f -not -name '.gitignore' -delete
```

Comment:
I suggest to use project's .gitignore so we can just rm this whole folder.

Reply:
-